recipe/meta.yaml

```diff
@@ -11,21 +11,18 @@ source:
   sha256: ff66c9142cae46317fb8b58e7acf59c963721e4f2a1505830cce99b57e78f091
 
 build:
-  number: 1
+  noarch: python
```

Comment:
FYI: This is the line that matters. It's then required to "rerender" to reconfigure the CI to only have one pipeline and not one-per-python-version-per-platform.

Reply:
Ok, thanks!